src/hpnmg/ProbabilityCalculator.h

```diff
@@ -116,6 +123,10 @@ namespace hpnmg {
                                                                 const vector<pair<string, map<string, float>>> &distributionsNormalized,
                                                                 char algorithm, int functioncalls, double &error);
 
+        double getProbabilityForUnionOfPolytopesUsingDirectMonteCarlo(vector<HPolytope<double>> polytopes,
```

Comment:
We should start adding more Documentation:

- Alternative to `getProbabilityForUnionOfPolytopesUsingMonteCarlo`
- computes the probability as integral of the combined pdf over the union of all polytopes in the vector.
- `distributions[i]` provides the pdf for the `i`-th dimension of the vector space. the pdf are assumed to be independent of each other.